### PR TITLE
Update NuGet feeds

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

**Comment by hal-ler** [comment]
Just a heads up, the build is starting to pull in rc3 parts from AspNetCore.

**Comment by troydai** [comment]
Ha? that is bad. Any idea now what package pulls in `rc3` packages?

**Comment by hal-ler** [comment]
I think AspNetCore just auto-updates to 1.0.0-rc3-20553.

**Comment by hal-ler** [comment]
Is there something like rc2-final that we could pin and be sure that it exists on the aspnet feed?

**Comment by troydai** [comment]
@hal-ler I can't answer that for obviously reason :). We're getting there.

File: .gitignore [hunk 1]
```diff
@@ -35,6 +35,7 @@ scripts/Omnisharp*
 
 # NuGet
 *.nuget.targets
+.packages/
```
Comment (troydai) [inline .gitignore]: this is not needed. I'll remove it.

**Pull request closed without merging**